### DISCUSSION TITLE
Is GitHub Copilot available for free to users who are neither students nor teachers? I'd like to know if there are any options for accessing GitHub Copilot without being affiliated with an educational institution.

### DISCUSSION THREAD

**Discussion by krishnauprety** [community/community#69685] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello everyone,

I'm curious about the availability of GitHub Copilot for users who don't have an affiliation with an educational institution as either students or teachers. While I understand that GitHub Copilot is often associated with educational benefits, I'd like to explore whether there are options for individuals who are not part of such institutions but are interested in using this powerful tool.

I appreciate any insights or information you can provide regarding this matter.

Thank you!

**Comment by santoshvandari** [comment, marked as answer]
Hello @krishnauprety,

GitHub Copilot is primarily available through GitHub Copilot subscriptions. While it's true that GitHub offers some educational benefits for students and teachers, GitHub Copilot is not exclusively limited to educational institutions.

Individuals and developers who are not affiliated with an educational institution can also subscribe to GitHub Copilot to access its features. GitHub Copilot offers both individual and team plans, and you can find the pricing and subscription details on the GitHub website.

To subscribe and start using GitHub Copilot, you can follow these general steps:

- Visit the GitHub Copilot subscription page on GitHub's website.

- Choose the subscription plan that best suits your needs, whether it's an individual plan or a team plan.

- Follow the prompts to set up your subscription and provide payment information.

Once your subscription is active, you'll be able to access and use GitHub Copilot in your development projects, whether you're a student, teacher, or an individual developer.

Please note that the availability and pricing of GitHub Copilot may be subject to change, so I recommend visiting the official GitHub Copilot website or contacting GitHub support for the most up-to-date information and options.

**Comment by kuroodo** [comment]
I want to add that you should look into Codeium. Since GitHub is going to be locking my account soon over 2FA, I cancelled my copilot subscription and had been looking for alternatives.

Codeium has so far been just as good as copilot (and better in various other aspects such as security). Best part is that it's free lol
